package.json

```diff
@@ -72,8 +72,8 @@
     "react-docgen-fixed": "^2.9.1",
     "react-dom": "16.6.3",
     "react-hot-loader": "^3.0.0-beta.6",
-    "react-router": "^2.6.0",
```

Comment:
why do we even need react-router in plasma?

Reply:
I am not sure we should

Reply:
@kangax it is not used anywhere in this repository -- I believe it can be removed as a dep

Reply:
removed